README.md

```diff
@@ -1,6 +1,7 @@
 [![Build Status](https://travis-ci.org/rainforestapp/rainforest-cli.png?branch=master)](https://travis-ci.org/rainforestapp/rainforest-cli)
```

Comment:
We should probably also move this to circle badge: https://circleci.com/docs/status-badges/